internal/cmd/gocdk/main.go

```diff
@@ -27,6 +27,13 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// generate_static converts the files in static/ into constants in a new
+// file,
+//go:generate go run generate_static.go
+
+// TODO(rvangent): Add a presubmit to Travis to ensure that the generated file
```

Comment:
I feel somewhat strongly that this check should be added before submitting this PR.

Reply:
Done.